src/index.js

```diff
@@ -346,17 +345,52 @@ function drawDailyIncreaseChart(sheetTrend) {
   })
 }
 
+function drawSparkLine(elementId, seriesData, maxConfirmedIncrease) {
```

Comment:
Thoughts on using something a bit more specific like `drawPrefectureTrendChart`? I think I mistakenly used the word [sparkline](https://en.wikipedia.org/wiki/Sparkline) in our previous chat.

Reply:
Good point, I'll just call it trend everywhere.